erdpy/accounts.py

```diff
@@ -22,7 +23,7 @@ def __init__(self,
                  pem_file: Optional[str] = None,
                  pem_index: int = 0,
                  key_file: str = "",
-                 pass_file: str = "",
+                 pass_file: Optional[str] = None,
```

Comment:
Perhaps pass the actual password, loaded in advance by the CLI components? (see the other comments)